plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -121,8 +138,10 @@
       - Target drive's backing file's data format.
       - Used only with clone
       - Use I(format=unspecified) and I(full=false) for a linked clone.
+      - If I(proxmox_default_behavior) is set to C(compatiblity) (the default value), this
+        option has a default of C(qcow2). Note that the default value of I(proxmox_default_behavior)
+        changes in community.general 4.0.0.
```

Comment:
Maybe mention here that the new default value (i.e. no default) is equivalent to `C(unspecified)`?

Reply:
https://github.com/ansible-collections/community.general/pull/850/commits/e77b1f615eff1af515df139ca80d56634827716c#diff-922781dde0b1a2ccb1f824f8230a668c297a77b436fe569448d4b38bf7324a1aL142-R140
Is this good enough?

Reply:
I find it a bit hard to understand. How about:
```
      - If I(proxmox_default_behavior) is set to C(compatiblity) (the default value), this
        option has a default of C(qcow2). If I(proxmox_default_behavior) is set to C(no_defaults),
        not specifying this option is equivalent to setting it to C(unspecified).
        Note that the default value of I(proxmox_default_behavior) changes in community.general 4.0.0.
```

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/850/commits/db0ed7c935773a82d7f24860c624dce95fbca274.